net/munge/Portfile

```diff
@@ -1,5 +1,4 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-# $Id$
```

Comment:
Karl is iterating this patch bit by bit (with Clemens) and it will finally get squashed it into one atomic commit and not left as fragmented as is. At least I think that's the plan. :)